andela/travel_tool_front
# #162640479 Place the 'Change Centre' Modal at the center of the page as per the design

#### What does this PR do?
Places the change center modal at the center of the page.

#### Description of Task to be completed?
-  ensure the change center modal is at the center of the page
- ensure the user role modal is at the center of the page

#### How should this be manually tested?
- Clone the repo
    `git clone https://github.com/andela/travel_tool_front.git`
 - Switch to the project directory
   `cd travel_tool_front`
 - Checkout the **ch-change-center-modal-162640479** branch
   `git checkout ch-change-center-modal-162640479`
 - Install dependencies and start the server
   - `make start` *if you have docker*
   - `yarn install` then `yarn start` *if you do not have docker*
 - Start the [backend](https://github.com/andela/travel_tool_back) of this project.
 - Navigate to [travela](http://travela-local.andela.com:3000/) in your browser and log in with your Andela account. 
 - Login as admin
- Navigate to [settings](http://travela-local.andela.com:3000/settings/roles) in your browser.
 - Click on one of the role names
 - You should see a list of users with that role.
 - Click on the *edit* button for one of the user.
- The **Change Center** modal should be displayed at the center of the page.

#### Any background context you want to provide?


#### What are the relevant pivotal tracker stories?
[#162640479](https://www.pivotaltracker.com/story/show/162640479, "Place the 'Change Centre' Modal at the center of the page as per the design")

#### Screenshots (if appropriate)
Before the changes the dialog was at 75% the width of the page
<img width="1680" alt="screenshot 2018-12-16 at 17 10 22" src="https://user-images.githubusercontent.com/25629064/50054692-86631880-0155-11e9-9877-1370d0b6147a.png">

Current the dialog is at the center of the page as required
<img width="1680" alt="screenshot 2018-12-16 at 17 08 59" src="https://user-images.githubusercontent.com/25629064/50054705-a561aa80-0155-11e9-8f3c-f4e8def97e82.png">
